style.css

```diff
@@ -343,3 +329,137 @@ p {
 .languages li {
   background: #f7f7f9;
 }
+
+.contact-container {
+  margin: 0 auto;
+  position: relative;
+  max-height: 800px;
+  margin-bottom: 50px;
+  background-color: white;
+}
+
+.form-container {
+  position: relative;
+  background-color: #6070ff;
+  margin-top: 177px;
+  height: 810px;
+  border-top-left-radius: 80px;
+  display: flex;
+  flex-direction: column;
+}
+
+.back-spot {
+  position: absolute;
+  z-index: 1;
+  top: 21px;
+  right: 0;
+}
+
+.form-bottom {
+  z-index: 10000;
+  position: absolute;
+}
+
```

Comment:
- [ ] Great job attempting previous suggestions on how to improve your contact section design but it appears the white space hasn't been fixed yet, looking at the design below:

![image](https://user-images.githubusercontent.com/99715640/230063503-9c14cd6d-fb97-413b-abff-4a17577be650.png)
  - [ ] It's very important to avoid including extraneous or hard-coding `height` and `width` values  for elements of your page to avoid unnecessary headaches.
  - [ ] To solve this problem at hand, kindly remove the `position: absolute;` that you have in `.form-bottom` styling property, and also remove the hard-coded `height: 810px;` in `.form-container`, and viola :muscle: everything should be fine.
  - [ ] [OPTIONAL] Your buttons do not have correct styling in both the `:hover` and `:active` states as recommended by the Figma design. Implementing this has the potential to improve user experience on your page as much as the UI. Here is the palette recommended by Figma, which you can inspect to get the appropriate CSS properties and values:
  
![image](https://user-images.githubusercontent.com/99715640/230067577-a5da8b9f-768b-405f-b289-34a8f08c1f3f.png)

Reply:
Ohh!! Gee. When I fixed the hard height it really shown in my vscode live server that the height was fixed. But after your review and checking in the chrome dev tool it shown that the error actually remains.

I have done exactly what you told and yeah it solves the error.
Optional: I have given the btn.hover more relevant color

And also gave margin bottom 8px at hr to replicate the figma design.

Thanks a lot @AbayomiOlaoye

Reply:
I have no more reviews left after now I am gonna submit. 😢 
I hope no more error are there.